bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/values/XmlSequence.java

```diff
@@ -628,6 +633,28 @@ public Object next() {
         };
     }
 
+    private void createXmlSequenceByCheckingAdjacentTextItems(List<BXml> values) {
+        this.children = new ArrayList<>();
+        boolean prev = false;
```

Comment:
Can we use a name that is better reflection of what this is tracking?

Reply:
Fixed with cfc81cfad46986467573c70a6c7a316ea085d2ae